python/pyarrow/_dataset.pyx

```diff
@@ -76,8 +76,54 @@ cdef class FileFormat:
 
 cdef class ParquetFileFormat(FileFormat):
 
+    cdef:
+        CParquetFileFormat* parquet_format
+
     def __init__(self):
```

Comment:
It might also be nice that the user can pass the different options as keywords to the constructor as well (similarly to how the csv options work)

Reply:
Will do